src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -133,5 +133,7 @@ public static class EnvironmentSettingNames
         public const string AppKind = "APP_KIND";
 
         public const string DrainOnApplicationStopping = "FUNCTIONS_ENABLE_DRAIN_ON_APP_STOPPING";
+        public const string Framework = "FRAMEWORK";
```

Comment:
As this only applies to the scenario you're covering, can you please be more specific with the constant name so others without this context would understand what they're for?

Reply:
Done

Reply:
I think Fabio asked for the constant name to be renamed here